modules/imgcodecs/test/test_grfmt.cpp

```diff
@@ -78,7 +78,9 @@ const string all_images[] =
     "readwrite/Bretagne2.jp2",
     "readwrite/Grey.jp2",
     "readwrite/Grey.jp2",
+    "readwrite/balloon.j2c",
```

Comment:
GDAL has issue with this file.
Need to update `notForGDAL` below.

Reply:
👍